src/Euclidean.jl

```diff
@@ -19,6 +19,14 @@ struct Euclidean{T<:Tuple} <: Manifold where {T} end
 Euclidean(n::Int) = Euclidean{Tuple{n}}()
 Euclidean(m::Int, n::Int) = Euclidean{Tuple{m,n}}()
 
+function representation_size(::Euclidean{Tuple{n}}, ::Type{T}) where {n,T<:Union{MPoint, TVector, CoTVector}}
```

Comment:
Oh, I think I get it now. If the array represents a point, you would use `representation_size (M, MPoint)` here to get the size. It would be nice to have that documented somewhere.